.github/workflows/build-release-alpine-nodejs.yml

```diff
@@ -34,22 +45,31 @@ jobs:
     - name: Upload alpine node.js
       uses: actions/upload-artifact@v4
       with:
-        name: alpine_nodejs_${{github.event.inputs.NodeVersion}}
-        path:  ${{runner.temp}}/alpine_node/node-${{github.event.inputs.NodeVersion}}-alpine-x64.tar.gz
+        name: alpine_nodejs_${{github.event.inputs.NodeVersion}}_${{matrix.arch}}
+        path: ${{runner.temp}}/alpine_node/node-${{github.event.inputs.NodeVersion}}-alpine-${{matrix.arch}}.tar.gz
 
   test:
-    name: Test node.js ${{github.event.inputs.NodeVersion}}
+    name: Test node.js ${{github.event.inputs.NodeVersion}} (${{matrix.arch}})
     needs: [build]
-    runs-on: ubuntu-latest
+    runs-on: ${{matrix.runner}}
+    strategy:
+      matrix:
+        include:
+          - arch: x64
+            runner: ubuntu-latest
+          # TODO enable running the tests on arm64 once actions/runner supports Alpine arm64
+          # https://github.com/actions/runner/pull/3665
+          #- arch: arm64
+          #  runner: ubuntu-24.04-arm
```

Comment:
Looks like running the tests will have to wait until https://github.com/actions/runner/pull/3665 is merged, because they obviously [fail](https://github.com/dennisameling/alpine_nodejs/actions/runs/12996727922/job/36249392462) due to the block in the `actions/runner` repo.